packages/react/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!<rootDir>/node_modules/'],
+  coverageReporters: ['text-summary', 'lcov'],
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  testEnvironment: 'jsdom',
+  testPathIgnorePatterns: ['lib'],
+  testRegex: '(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$',
+  transform: {
+    '.(ts|tsx)': 'ts-jest',
+  },
+};
```

Comment:
1. should use `.ts`
2. some props are not necessary

Reply:
<img width="1241" alt="image" src="https://user-images.githubusercontent.com/10594507/155503575-2540deef-2d4e-423c-aa73-441b90893a88.png">
If `js/react` provide `Provider`, it should contain `.tsx` files?

Reply:
Updated. PTAL.